datascience-notebook/onbuild/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM  jupyter/datascience-notebook
+
+ONBUILD COPY requirements.txt /home/jovyan/work
```

Comment:
Probably can just replaces this with `ONBUILD COPY . /home/jovyan/work` given requirements is in `.`.